src/main/scala/is/hail/io/vcf/ExportVCF.scala

```diff
@@ -44,10 +43,8 @@ object ExportVCF {
       case TString(_) =>
         sb.append(TString.loadString(m, offset))
       case TCall(_) =>
-        val p = Genotype.gtPair(m.loadInt(offset))
-        sb.append(p.j)
-        sb += '/'
-        sb.append(p.k)
+        val c = m.loadInt(offset)
+        Call.addToStringBuilder(c, Call.vcfString, sb)
```

Comment:
The `vcfString` method allocates a bunch of stuff en route to making a string and feeding it into the builder.

Maybe Call.addToStringBuilder should take `f: StringBuilder => Unit`?